lib/introspector.js

```diff
@@ -221,7 +221,32 @@ const parseRest = lines => {
   return result;
 };
 
+const findInBounds = (str, startStr, endStr) => {
```

Comment:
I'd prefer to add `offset = 0` as the fourth parameter to avoid doing `slice`s below.